jscomp/bsb/bsb_ninja_targets.ml

```diff
@@ -210,46 +186,37 @@ let output_build
    Buffer.add_string buf ")))";
   end;
   Buffer.add_string buf "(deps (:inputs ";
-  Ext_list.iter inputs (fun s ->   Buffer.add_string buf Ext_string.single_space ; Buffer.add_string buf (Filename.basename s));
+  Ext_list.iter inputs (fun s ->
+    Buffer.add_string buf Ext_string.single_space;
+    Buffer.add_string buf (Filename.basename s));
   Buffer.add_string buf ") ";
-  if implicit_deps <> [] then
-    begin
-      Ext_list.iter implicit_deps (fun s -> Buffer.add_string buf Ext_string.single_space; Buffer.add_string buf (Filename.basename s))
-    end
-  ;
-  if order_only_deps <> [] then
```

Comment:
we used to have a `Module.depends` rule generated by the alias `@bsb_depends` so that we could track module dependencies (which are only discovered by ocamldep after we generate the dune files).

with the dune action plugin stuff that's not needed anymore.